src/serializers/resource.js

```diff
@@ -7,8 +7,7 @@ const SmartFieldsValuesInjector = require('../services/smart-fields-values-injec
 const Schemas = require('../generators/schemas');
 const logger = require('../services/logger');
 
-function ResourceSerializer(Implementation, model, records, integrator, opts, meta,
-  fieldsSearched, searchValue, fieldsPerModel) {
+function ResourceSerializer(Implementation, model, records, integrator, opts, fieldsSearched,searchValue, fieldsPerModel) {
```

Comment:
The ResourceSerializer is a "public" function exposed for the Smart Collection implementations for example. If people cannot add meta anymore, they won't have the count anymore.

What would happen to the people using the current signature in production if they upgrade to this version?